src/content/test/parsing/keyboard-bookmarklet-instructions.tsx

```diff
@@ -1,157 +1,164 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { NewTabLink } from '../../../common/components/new-tab-link';
-import { create, React } from '../../common';
+import { createWithTitle, React } from '../../common';
 
-export const keyboardBookmarkletInstructions = create(({ Markup }) => (
-    <>
-        <Markup.Title>Using a keyboard to add the Nu HTML Checker bookmarklets</Markup.Title>
-        <h2>Google Chrome instructions</h2>
-        <ol>
-            <li>
-                Open the Chrome <Markup.Term>Bookmarks</Markup.Term> manager:
-                <ol type="a">
-                    <li>
-                        In Chrome, type <Markup.Term>Ctrl + Shift + o</Markup.Term> or <Markup.Term>⌘ + Option + b</Markup.Term>.
-                    </li>
-                    <li>
-                        A new browser tab called <Markup.Term>Bookmarks</Markup.Term> will open.
-                    </li>
-                </ol>
-            </li>
+export const keyboardBookmarkletInstructions = createWithTitle(
+    {
+        pageTitle: '',
```

Comment:
This seems off

Reply:
This is not info and examples content, so I have undone all changes to the file. Looks like it was the only one that didn't belong. Good catch!